topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -881,29 +870,25 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    2. `brand` to something fun! (E.g. 🧬🔬🚀)
 >    3. `database_connection` to point to the database you setup earlier (`postgresql:///galaxy?host=/var/run/postgresql`).
 >    4. `file_path` to a place to store data, `/data` is fine for this lesson which sets up a single-node Galaxy. If you have separate compute machines, this will normally need to be storage shared between the Galaxy node and compute nodes.
->    5. `check_migrate_tools` must be set to `false` due to a new installation of Galaxy.
->    6. `tool_data_path` to {% raw %}`{{ galaxy_mutable_data_dir }}/tool-data`{% endraw %}, so that when tools are installed, due to privilege separation, this will happen in a directory Galaxy can actually write into.
->    7. `object_store_store_by` to `uuid`, this is a better way of storing files that will ensure better filesystem balancing than the older system.
->    8. `id_secret` to {% raw %}`{{ vault_id_secret }}`{% endraw %}, we'll define this variable next but it will be used to encode the IDs used in Galaxy URLs and for securing session cookies.
+>    5. `tool_data_path` to {% raw %}`{{ galaxy_mutable_data_dir }}/tool-data`{% endraw %}, so that when tools are installed, due to privilege separation, this will happen in a directory Galaxy can actually write into.
+>    6. `id_secret` to {% raw %}`{{ vault_id_secret }}`{% endraw %}, we'll define this variable next but it will be used to encode the IDs used in Galaxy URLs and for securing session cookies.
 >
 >    {% raw %}
 >    ```diff
 >    --- a/group_vars/galaxyservers.yml
 >    +++ b/group_vars/galaxyservers.yml
->    @@ -26,3 +26,14 @@ galaxy_force_checkout: true
+>    @@ -26,3 +26,12 @@ galaxy_force_checkout: true
 >     miniconda_prefix: "{{ galaxy_tool_dependency_dir }}/_conda"
->     miniconda_version: 4.7.12
->     miniconda_manage_dependencies: false
+>     miniconda_version: 4.12.0
+>     miniconda_channels: ['conda-forge', 'defaults']
 >    +
 >    +galaxy_config:
 >    +  galaxy:
 >    +    brand: "🧬🔬🚀"
 >    +    admin_users: admin@example.org
 >    +    database_connection: "postgresql:///galaxy?host=/var/run/postgresql"
 >    +    file_path: /data
->    +    check_migrate_tools: false
 >    +    tool_data_path: "{{ galaxy_mutable_data_dir }}/tool-data"
->    +    object_store_store_by: uuid
```

Comment:
is this default now or so?

Reply:
Acutally - I thought I verified that it was, but now I'm not sure:

>          What Dataset attribute is used to reference files in an ObjectStore implementation,
>          this can be 'uuid' or 'id'. The default will depend on how the object store is configured,
>          starting with 20.05 Galaxy will try to default to 'uuid' if it can be sure this
>          is a new Galaxy instance - but the default will be 'id' in many cases. In particular,
>          if the name of the directory set in <file_path> is `objects`, the default will be set
>          to 'uuid', otherwise it will be 'id'.